src/test/resources/mocks/management/companies/list-companies-response.json

```diff
@@ -0,0 +1,29 @@
+{
+  "_links": {
+    "first": {
+      "href": "http://localhost:8080/configurationapi/v1/companies?pageNumber=1&pageSize=10"
```

Comment:
Minor, we could use here the real `Management API` endpoint of the `TEST` environment for the test samples.